content/applications/inventory_and_mrp/inventory/shipping/setup/sendcloud_shipping.rst

```diff
@@ -0,0 +1,221 @@
+After the Sendcloud account is set up and configured, it's time to configure the Odoo database.
+To get started, go to Odoo's :guilabel:`Apps` module,  search for the :guilabel:`Sendcloud
+Shipping` integration, and install it.
+
+.. image:: sendcloud_shipping/sendcloud-mod.png
+   :align: center
+   :alt: Sendcloud Shipping module in the Odoo Apps module.
+
+Sendcloud shipping connector configuration
+------------------------------------------
+
+Once installed, activate the :guilabel:`Sendcloud Shipping` module in :menuselection:`Inventory -->
+Configuration --> Settings`. The :guilabel:`Sendcloud Connector` setting is found under the
+:guilabel:`Shipping Connectors` section.
+
+After activating the :guilabel:`Sendcloud Connector`, click on the :guilabel:`Sendcloud Shipping
+Methods` link below the listed connector. Once on the :guilabel:`Shipping Methods` page, click
+:guilabel:`Create`.
+
+.. tip::
+   :guilabel:`Shipping Methods` can also be accessed by going to :menuselection:`Inventory -->
+   Configuration --> Delivery --> Shipping Methods`.
+
+Fill out the following fields in the :guilabel:`New Shipping Method` form:
+
+- :guilabel:`Shipping Method`: type `Sendcloud DPD`.
+- :guilabel:`Provider`: select :guilabel:`Sendcloud` from the drop-down menu.
+- :guilabel:`Delivery Product`: set the product that was configured for this shipping method or
+  create a new product.
+- In the :guilabel:`SendCloud Configuration` tab, enter the :guilabel:`Sendcloud Public Key`.
+- In the :guilabel:`SendCloud Configuration` tab, enter the :guilabel:`Sendcloud Secret Key`.
+- Manually :guilabel:`Save` the form by clicking the cloud icon next to the :guilabel:`Shipping
+  Methods / New` breadcrumbs.
+
+After configuring and saving the form, follow these steps to load the shipping products:
+
+- In the :guilabel:`SendCloud Configuration` tab of the :guilabel:`New Shipping Method` form, click
+  on the :guilabel:`Load your SendCloud shipping products` link.
+- Select the shipping products the company would like to use for deliveries and returns.
+- Click :guilabel:`Select`.
+
+.. example::
+   Sample Sendcloud shipping products configured in Odoo:
+
+   | **Delivery:**
+   | :guilabel:`Shipping Product` - DPD Home 0-31.5kg
+   | :guilabel:`Carrier` - DPD
+   | :guilabel:`Minimum Weight` - 0.00
+   | :guilabel:`Maximum Weight` - 31.50
+
+   :guilabel:`Countries` - Austria, Belgium, Bosnia and Herzegovina, Bulgaria, Croatia, Czech
+   Republic, Denmark, Estonia, Finland, France, Germany, Greece, Hungary, Iceland, Ireland, Italy,
+   Latvia, Liechtenstein, Lithuania, Luxembourg, Monaco, Netherlands, Norway, Poland, Portugal,
+   Romania, Serbia, Slovakia, Slovenia, Spain, Sweden, Switzerland
+
+   | **Return:**
+   | :guilabel:`Return Shipping Product`- DPD Return 0-20kg
+   | :guilabel:`Return Carrier` - DPD
+   | :guilabel:`Return Minimum Weight` - 0.00
+   | :guilabel:`Return Minimum Weight` - 20.00
+   | :guilabel:`Return Countries` - Belgium, Netherlands
```

Comment:
Same here— would format everything like previous `example` tag.
Notice the `guilabel` formatting for `DELIVERY` and `RETURN` as this is how it shows up on the GUI.

```suggestion
.. example::
   Sample Sendcloud shipping products configured in Odoo:

   | :guilabel:`DELIVERY`
   | :guilabel:`Shipping Product`: `DPD Home 0-31.5kg`
   | :guilabel:`Carrier`: `DPD`
   | :guilabel:`Minimum Weight`: `0.00`
   | :guilabel:`Maximum Weight`: `31.50`

   :guilabel:`Countries`: `Austria` `Belgium` `Bosnia` `Herzegovina` `Bulgaria` `Croatia` `Czech`
   `Republic` `Denmark` `Estonia` `Finland` `France` `Germany` `Greece` `Hungary` `Iceland`
   `Ireland` `Italy` `Latvia` `Liechtenstein` `Lithuania` `Luxembourg` `Monaco` `Netherlands`
   `Norway` `Poland` `Portugal` `Romania` `Serbia` `Slovakia` `Slovenia` `Spain` `Sweden`
   `Switzerland`

   | :guilabel:`RETURN`
   | :guilabel:`Return Shipping Product`: `DPD Return 0-20kg`
   | :guilabel:`Return Carrier`: `DPD`
   | :guilabel:`Return Minimum Weight`: `0.00`
   | :guilabel:`Return Minimum Weight`: `20.00`
   | :guilabel:`Return Countries`: `Belgium` `Netherlands`
```

Reply:
@StraubCreative 
CC: @samueljlieber 

The changes in this commit look great! The examples are really clear now. Thanks for making the changes 👍 

[635cad7](https://github.com/odoo/documentation/pull/2864/commits/635cad7caedaf3c90b5158fead76983fc4f9e464)

This commit is good to push up and send to DR. 

Thanks for all your work on this. Looking forward to seeing it published. 

👍